File: generator_cpf.py
from random import randint

nums = ''.join(list(map(lambda n: str(n), [randint(1,10) for x in range(9)])))
comp1 = [x for x in range(10,1,-1)]
comp2 = [x for x in range(11,1,-1)]

def calc(cpf):
    sum = 0

    try:
        for n in range(len(cpf)):
            if len(cpf) < 10:
                sum += int(cpf[n]) * comp1[n]
            else:
                sum += int(cpf[n]) * comp2[n]
    except:
        global nums
        nums = ''.join(list(map(lambda n: str(n), [randint(1,10) for x in range(9)])))
        calc(nums)

    sum = sum * 10 % 11

    return str(sum) if sum < 10 else '0'

final_digits = calc(nums)
final_digits += calc(nums+final_digits)
print(nums + final_digits)

File: generator_cpf.js
let digits_cpf = String(Math.random()).split('').splice(9).join('')
let n1 = [10,9,8,7,6,5,4,3,2]
let n2 = [11,10,9,8,7,6,5,4,3,2]

let calc_nums = (cpf) => {  // Sim, eu criei isto.
    let sum = 0;

    for(let i = 0; i < digits_cpf.length; i++)
        sum += digits_cpf[i] * (cpf.length == 9 ? n1[i] : n2[i]);

    return (sum*10 % 11) == 10 ? 0 : sum*10 % 11;
}

digits_cpf += calc_nums(digits_cpf);
digits_cpf += calc_nums(digits_cpf);

console.log(digits_cpf)
